facebook/delphiFacebook/R/contingency_aggregate.R

```diff
@@ -95,6 +96,17 @@ produce_aggregates <- function(df, aggregations, cw_list, params) {
     if ( length(dfs_out) != 0 ) {
       df_out <- dfs_out %>% reduce(full_join, by=agg_group, suff=c("", ""))
       write_contingency_tables(df_out, params, geo_level, agg_group)
+      
+      for (theme in names(THEME_GROUPS)) {
```

Comment:
Where is `THEME_GROUPS` defined? I don't see it elsewhere in this PR

Reply:
It's at the very top of `contingency_indicators.R`. The diff is collapsed because of its size.

Reply:
> The diff is collapsed because of its size.

ugh, thanks GitHub